draft-ietf-spice-sd-cwt.md

```diff
@@ -721,61 +738,49 @@ IANA is requested to add the following entries to the CWT claims registry (https
 
 The following completed registration template per RFC8392 is provided:
 
-Claim Name: redacted_claim_keys
-Claim Description: Redacted Claim Keys in a map.
-JWT Claim Name: _sd
-Claim Key: TBD5 (request assignment 65556)
-Claim Value Type(s): array of bstr
-Change Controller: IETF
-Specification Document(s): RFC XXXX
-
-### redacted_claim_element
-
-The following completed registration template per RFC8392 is provided:
-
-Claim Name: redacted_claim_element
-Claim Description: Redacted element of an array
-JWT Claim Name: ...
-Claim Key: TBD6 (request assignment 65555)
-Claim Value Type(s): array of bstr
-Change Controller: IETF
-Specification Document(s): RFC XXXX
+* Claim Name: redacted_claim_keys
+* Claim Description: Redacted Claim Keys in a map.
+* JWT Claim Name: _sd
+* Claim Key: TBD5 (request assignment -65536)
+* Claim Value Type(s): array of bstr
+* Change Controller: IETF
+* Specification Document(s): RFC XXXX
```

Comment:
For this stuff, I recommend something like this:

```
Claim Name:
: redacted_claim_keys

Claim Description:
...

Specification Document(s):
: RFC XXXX
{: spacing="compact"}
```

Reply:
Thanks. with compact and bold added, this is certainly better, but when we get to the Additional information: field with 3 subfields, that syntax just doesn't know how to handle it.  It is also way too verbose in the source. I'll stick with the bullet items for this document, but use the compact definition rendering (with bold) in future docs that don't have multiple indent levels.